examples/02_market_data.py

```diff
@@ -0,0 +1,166 @@
+"""
+Market Data: Exploring Available Markets and Price Information
+
+This example demonstrates:
+1. Discovering available currencies and instruments
+2. Finding trading opportunities (bid-ask spreads)
+3. Monitoring market conditions across assets
+4. Comparing perpetual vs option markets
+5. Using cached vs fresh data
+
+Prerequisites:
+- None! Market data is public and doesn't require authentication
+"""
+
+from decimal import Decimal
+from pathlib import Path
+
+from derive_client import HTTPClient
+from derive_client.data_types import InstrumentType
+
+# Setup
+env_file = Path(__file__).parent.parent / ".env.template"
+client = HTTPClient.from_env(env_file=env_file)
+client.connect()
+
+print("=" * 60)
+print("1. DISCOVER AVAILABLE MARKETS")
+print("=" * 60)
+
+# Get all currencies
+currencies = client.markets.get_all_currencies()
+print(f"\nAvailable currencies: {len(currencies)}")
+for curr in currencies[:5]:  # Show first 5
+    print(f"  {curr.currency}: ${curr.spot_price:.2f}")
+print("  ...")
+
+# Get instruments for ETH
+eth_perps = client.markets.get_instruments(
+    currency="ETH",
+    expired=False,
+    instrument_type=InstrumentType.perp,
+)
+print(f"\nETH perpetuals: {len(eth_perps)}")
+for perp in eth_perps:
+    print(f"  {perp.instrument_name}")
+
+eth_options = client.markets.get_instruments(
+    currency="ETH",
+    expired=False,
+    instrument_type=InstrumentType.option,
+)
+print(f"\nETH options: {len(eth_options)} active")
+
+print("\n" + "=" * 60)
+print("2. FIND TIGHT SPREADS (Best Trading Conditions)")
+print("=" * 60)
+
+# Compare spreads across major perpetuals
+perp_names = ["ETH-PERP", "BTC-PERP"]
+spreads = []
+
+print("\nBid-Ask Spreads (lower = better liquidity):")
+for instrument in perp_names:
+    ticker = client.markets.get_ticker(instrument_name=instrument)
```

Comment:
same comment as above, not sure if using the new `get_tickers`, but should move over to that.

Reply:
Is this endpoint going to be totally removed?
Seems a little odd to not have get_ticker anymore?